suite-native/bluetooth/src/components/BluetoothDeviceCard.tsx

```diff
@@ -45,7 +44,7 @@ export const BluetoothDeviceCard = ({
     return (
         <Card style={applyStyle(cardStyle)}>
             <Box alignItems="center">
-                <DeviceImage deviceModel={deviceModel} size="small" />
+                <DeviceBackplateImage color={deviceColor} />
```

Comment:
Wouldn’t it be better to keep it in the `DeviceImage` component and just add the images to the array, like `T3W1-color`, `T3W1-anotherColor`, etc.? It could be usefull if we need backplate images for other models.

Reply:
It doesn't support any other models because we don't have images for any other models at the moment. Once we have some other BT devices, we can extend the component and/or move it elsewhere. I don't see any point in implementing support for something that might never occur.

Reply:
Seeing the glass half empty huh? :D But yeh, you have solid points. Thanks for the discussion 💪

Reply:
No, years of coding experience. 😉 I've seen too many lines of code written just because "we might use it one day" and no one did after 10 or 15 years. The only outcome was more complex codebase, nothing more.